src/components/common/AuthProtectedRoute.jsx

```diff
@@ -1,14 +1,14 @@
 import { Navigate } from 'react-router';
-import { useSelector } from 'react-redux';
+import useStorageValue from '../../hooks/useStorageValues';
 
 const AuthProtectedRoute = ({ children }) => {
-    const user = useSelector((state) => state.user);
+    const token = useStorageValue('authToken');
```

Comment:
do we have requests on all pages?
i would say that it's not proper way to check if user is authorized or not

Reply:
I did this check through local storage so that when the site is opened again, the user is authorized.

Reply:
if it's an expired token and no requests on the page it wouldn't redirect me to /login

Reply:
I don't know how to implement this logic so that it is only on the backend (it was in the task). If you know how, I would appreciate it.
Currently, the only page accessible to authorized users has a request that will log the user out if the refresh token has expired.